### PR TITLE
firefox needs to use merge strategy as of version 47+

### LINKED ISSUE #40: Firefox stops working with new Marionette driver (Firefox >=47)
Firefox introduced a new selenium driver (Marionette) which is required to run selenium tests since version 47.

Unfortunately they changed the way how Firefox takes screenshots. Until this change we always got whole page screenshots, but with Marionette we just get viewport screenshots.

We need to investigate
- ~~if the new screenshot mechanism is a bug? I think~~ it's intended to fulfill the [spec](http://w3c.github.io/webdriver/webdriver-spec.html#dfn-take-screenshot)
- how can we detect Marionette?

Related:
- [Marionette WebDriver](https://developer.mozilla.org/en-US/docs/Mozilla/QA/Marionette/WebDriver)
- [WebDriver Implementation Status](https://developer.mozilla.org/en-US/docs/Mozilla/QA/Marionette/WebDriver/status)
- mozilla/geckodriver#265

Currently, I see two solutions to fix this:
- detect marionette and stitch screenshots like chrome
- stitch screenshots always for firefox like chrome (changes also screenshot behavior of firefox <= 46)

### PR DESCRIPTION
**Pull request by wouterverweirder**
Firefox no longer creates a full document screenshot and needs to use the merge strategy to generate a full document screenshot.

This pull request should fix #40

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "wdio-screenshot",
-  "version": "0.3.0",
+  "version": "0.3.1",
```
Comment (zinserjan) [inline package.json]: Can you please undo this? [np](https://github.com/sindresorhus/np) does this automatically when publishing a new release,

**Comment by zinserjan** [comment]
:thumbsup: Thanks for your PR.

My original idea was to support the fullpage screenshots of the older firefox versions, too. But I think it's not really worth it, cause it makes the code just more complicated...

**Comment by wouterverweirder** [comment]
Removed that version bump from my package.json & updated the commit :)

File: package.json (hunk 1 above)
Comment (wouterverweirder) [inline package.json]: Done!

**Comment by zinserjan** [comment]
Cool, LGTM :)

**Pull request merged**